src/Microsoft.VisualStudio.LanguageServices.Razor/Legacy/LegacyTemplateEngineFactoryService.cs

```diff
@@ -3,8 +3,11 @@
 
```

Comment:
Make sure not to commit this file in the initial push.

Reply:
Yeah will undo. Here lies my attempt to do this in a non-stateful way.